mk/update-travis-yml.py

```diff
@@ -61,7 +61,8 @@
 
 modes = [
     "DEBUG",
-    "RELWITHDEBINFO"
+    "RELWITHDEBINFO",
+    "NODEFAULTFEATURES"
```

Comment:
Nit: add a trailing comma here, so that next time you don't have to change this line if you're adding more.